blond/__init__.py

```diff
@@ -10,8 +10,12 @@
         libblond = ctypes.CDLL(os.path.join(
             basepath, 'cpp_routines/libblond.so'))
     elif ('win' in sys.platform):
-        libblond = ctypes.CDLL(os.path.join(
-            basepath, 'cpp_routines\\libblond.dll'))
+        dllpath = os.path.join(basepath, 'cpp_routines')
+        if 'add_dll_directory' in dir(os):
+            os.add_dll_directory(dllpath)
+            libblond = ctypes.CDLL(os.path.join(dllpath, 'libblond.dll'), winmode=0)
+        else:
+            libblond = ctypes.CDLL(os.path.join(dllpath, 'libblond.dll'))
```

Comment:
Either:
`if hasattr('add_dll_directory', os):`

Or:
```python
try:
    os.add_dll_directory(dllpath)
except AttributeError: #Might be a different exception
    libblond = ctypes.CDLL(os.path.join(dllpath, 'libblond.dll'))
else:
    libblond = ctypes.CDLL(os.path.join(dllpath, 'libblond.dll'), winmode=0)
```

Reply:
Thanks for the comments on the style, will implement the changes. Just one small thing: hasattr is used the other way around. `hasattr(object, name)` so it'd be
`if hasattr(os, 'add_dll_directory'):`